docs/src/_getting-started/installation.md

```diff
@@ -28,6 +28,8 @@ Currently Python 3.7, 3.8, 3.9, and 3.10 are supported. Some plugins do not yet
 <div class="termy">
 
 ```console
+$ python -m pip install --upgrade pip
+Requirement already satisfied.
 $ pip install "meltano"
```

Comment:
There should be consistency between these two `pip` invocations. Using `pip` on its own should be safe, or at least no safer than `python -m pip` (compared to `python3 -m pip` :unamused:).

```suggestion
$ pip install --upgrade pip
Requirement already satisfied.
$ pip install "meltano"
```